content/webapp/components/ImageGallery/ImageGallery.test.tsx

```diff
@@ -0,0 +1,87 @@
+import * as prismic from '@prismicio/client';
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ThemeProvider } from 'styled-components';
+
+import { AppContextProvider } from '@weco/common/views/components/AppContext/AppContext';
+import theme from '@weco/common/views/themes/default';
+
+import ImageGallery from './';
+
+window.HTMLElement.prototype.scrollIntoView = jest.fn(); // scrollIntoView is not in JSDOM: https://stackoverflow.com/a/60225417
+
+const image = {
+  contentUrl:
+    'https://cardigan.wellcomecollection.org/images/reading-room-3200x1800.jpg',
+  width: 3200,
+  height: 1800,
+  alt: 'an image with some alt text',
+  tasl: {
+    title: 'The title of the image',
+    author: 'The author',
+    sourceName: 'Wellcome Collection',
+    sourceLink: 'https://wellcomecollection.org/works',
+  },
+};
+const captionedImage = () => ({
+  image,
+  caption: [
+    {
+      type: 'paragraph',
+      text: 'Etiam pellentesque dui tellus, quis dictum turpis blandit id. Etiam.',
+      spans: [],
+    },
+  ] as prismic.RichTextField,
+  hasRoundedCorners: false,
+});
+
+const images = [captionedImage()];
+
+const renderComponent = () => {
+  const ImageGalleryExample = () => {
+    return (
+      <ThemeProvider theme={theme}>
+        <AppContextProvider>
+          <ImageGallery
+            id="test"
+            isStandalone={false}
+            isFrames={false}
+            items={images}
+          />
+        </AppContextProvider>
+      </ThemeProvider>
+    );
+  };
+  render(<ImageGalleryExample />);
+};
+
+describe('ImageGallery', () => {
+  it('should have an unfocused button on initial render', () => {
+    renderComponent();
+    const openButton = screen.getByRole('button', { name: '1 images' });
+    return expect(openButton).not.toHaveFocus();
+  });
+
+  it('should focus the close button when opened', async () => {
+    renderComponent();
+    const openButton = screen.getByRole('button', { name: '1 images' });
```

Comment:
We should pass the text in our pluralize function! (1 image/s)
not necessary for this PR though

Reply:
I was surprised that we hadn't already, but I suppose the reality is that there wouldn't be an image gallery with one image

Reply:
I think the helper is newer than the component tbf